<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllGamesPage</class>
 <widget class="QWidget" name="AllGamesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" rowspan="2">
    <widget class="QScrollArea" name="filtersScrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="filtersScrollAreaContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>171</width>
        <height>531</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="filtersScrollAreaLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QCheckBox" name="discountedOnlyCheckbox">
         <property name="text">
          <string>Show only discounted</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="hideDlcCheckbox">
         <property name="text">
          <string>Hide DLCs and extras</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QStackedWidget" name="contentStack">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="nothingFoundPage">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="nothingFoundLabel">
         <property name="text">
          <string>Nothing found :(</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="resultsPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="resultsHeader">
         <item>
          <widget class="QLabel" name="pagesLabel">
           <property name="text">
            <string>%d of %d</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="resultsHeaderSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="sortOrderLabel">
           <property name="text">
            <string>Sort by:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sortOrderComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>Bestselling (recently)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bestselling (all time)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Price (from lowest)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Price (from highest)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Discount (from highest)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Title (A to Z)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Title (Z to A)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Release date (from newest)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Release date (from oldest)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Rating (from highest)</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="gridModeButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/icons/table-cells.svg</normaloff>:/icons/table-cells.svg</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="listModeButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/icons/list.svg</normaloff>:/icons/list.svg</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QScrollArea" name="resultsScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="resultsScrollAreaContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>66</width>
            <height>16</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="resultsFooter">
         <item>
          <spacer name="leftResultsFooterSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="prevResultsButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/icons/angle-left.svg</normaloff>:/icons/angle-left.svg</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextResultsButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/icons/angle-right.svg</normaloff>:/icons/angle-right.svg</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="rightResultsFooterSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="errorPage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="errorLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Uh-oh, something bad happened. Retry?</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="retryButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Retry</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/rotate.svg</normaloff>:/icons/rotate.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="loadingPage"/>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="header" native="true">
     <layout class="QHBoxLayout" name="headerLayout">
      <item>
       <widget class="QLabel" name="totalLabel">
        <property name="text">
         <string>Showing %d games</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="placeholderText">
         <string>Search store by title, publisher or tag</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
